etcdserver/api/membership/cluster.go

```diff
@@ -752,6 +786,36 @@ func (c *RaftCluster) IsLocalMemberLearner() bool {
 	return localMember.IsLearner
 }
 
+// DowngradeInfo returns the capability status of the cluster
```

Comment:
DowngradeInfo returns the downgrade status of the cluster?

Reply:
Sgtm.